editor/src/clj/editor/camera_editor.clj

```diff
@@ -34,19 +35,22 @@
            [javax.vecmath Matrix4d Vector3d Vector4d Quat4d]))
 
 (set! *warn-on-reflection* true)
+(set! *unchecked-math* :warn-on-boxed)
 
 (def camera-icon "icons/32/Icons_20-Camera.png")
 
-(defn- get-camera-mesh-lines-json [path]
+(def camera-mesh-json "meshes/camera-preview-edge-list.json")
+
+(defn- read-json-resource [path]
   (with-open [reader (io/reader (io/resource path))]
     (json/read reader)))
-(def camera-mesh-lines
-  (let [camera-json-path "camera-preview-edge-list.json"
-        camera-json-data (first (get-camera-mesh-lines-json camera-json-path))
-        camera-edge-list (concat (second camera-json-data))
-        camera-mesh-lines-v4 (map (fn [p]
-                                    (Vector4d. (get p 0) (get p 1) (get p 2) 1))
-                                  camera-edge-list)]
+
+(def ^:private camera-mesh-lines
+  (let [camera-name+edge-list (first (read-json-resource camera-mesh-json)) ; First mesh name + edge-list.
+        camera-edge-list (second camera-name+edge-list)
+        camera-mesh-lines-v4 (mapv (fn [p]
+                                     (Vector4d. (p 0) (p 1) (p 2) 1.0))
+                                   camera-edge-list)]
```

Comment:
* Made the var private to this file.
* Renamed some stuff for clarity.
* Use `(p 0)` syntax to access Clojure vector elements.
* Use `mapv` instead of `map` to ensure the transformation happens immediately - not when first iterated over.